extensions/amp-lightbox-viewer/0.1/service/lightbox-manager-impl.js

```diff
@@ -161,6 +161,24 @@ export class LightboxManager {
   }
 
   /**
+   * The function is simplified for testing now.
+   * Get the description for single lightboxed item.
+   * @param {!Element} element
+   * @return {?string}
+   */
+  getDescription(element) {
+    const aria = element.getAttribute('aria-describedby');
```

Comment:
`aria-describedby` is actually an ID of another element so you want.

if(aria) {
  const descriptionElement = document.getElementById(aria);
 if(!descriptionElement) {
  return;
 }
 return descriptionElement.textContent;
}